csharp/private/repositories.bzl

```diff
@@ -209,6 +226,14 @@ def _net_workspace():
         build_file = "@d2l_rules_csharp//csharp/private:frameworks/netstandard20.BUILD",
     )
 
+    nuget_package(
+        name = "NetStandard.Library.Ref",
```

Comment:
I'm surprised we're getting away with naming workspaces with dots... https://docs.bazel.build/versions/0.29.1/skylark/lib/globals.html#workspace

>  Workspace names should be a Java-package-style description of the project, using underscores as separators, e.g., github.com/bazelbuild/bazel should use com_github_bazelbuild_bazel. Names must start with a letter and can only contain letters, numbers, and underscores.

Reply:
:man_shrugging:

Reply:
Yeah, basically.